components/02-elements/breadcrumbs/_breadcrumbs.scss

```diff
@@ -41,12 +40,12 @@ $breadcrumbs-font-family-active: $font-family-base-bold !default;
     &__link {
         display: flex;
         align-items: center;
-        padding: 0 $breadcrumbs-padding;
-        font-family: $breadcrumbs-font-family;
-        color: $breadcrumbs-color;
+        padding: 0 $breadcrumbs__padding;
+        font-family: $breadcrumbs__font-family;
+        color: $breadcrumbs__color;
 
         &:hover {
-            color: $breadcrumbs-color-active;
+            color: $breadcrumbs__color--active;
         }
         &:active,
         &:focus {
```

Comment:
line 55 - if you use `outline: none`, please put it in variable using global variable, b'c we have to change it